jenkins_jobs/run_host_os_admin_commands/script.sh

```diff
@@ -0,0 +1,26 @@
+VERSIONS_REPO_DIR=$(basename $VERSIONS_REPO_URL .git)
+
+
+# Fetch pull requests in case this job was triggered by one
+git clone $VERSIONS_REPO_URL $VERSIONS_REPO_DIR --no-checkout
+pushd $VERSIONS_REPO_DIR
```

Comment:
The path to the versions repository is now `workspace/repositories/versions`.
See https://github.com/open-power-host-os/builds/pull/191/commits/38fa4129603377d5a632025e0367d1bfb9200c60#diff-4ef649d5da4e7f2fba8387f34f84d63fR21